node/messages/Cargo.toml

```diff
@@ -46,6 +45,13 @@ version = "1"
 [dependencies.serde]
 version = "1"
 
+[dependencies.serde_json]
+version = "1"
+
+[dependencies.rand]
+version = "0.8.5"
+features = ["small_rng"]
```

Comment:
why are we using `small_rng` here? it should only be used in tests, as it's insecure